src/plugins/data/common/search/expressions/filters_to_ast.test.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { filtersToAst } from './filters_to_ast';
+
+describe('interpreter/functions#filtersToAst', () => {
+  const normalFilter = {
+    meta: { negate: false, alias: '', disabled: false },
+    query: { test: 'something' },
+  };
+  const negatedFilter = {
+    meta: { negate: true, alias: '', disabled: false },
+    query: { test: 'something' },
+  };
+
+  it('returns an object with the correct structure', () => {
```

Comment:
über nit: it is usually better to not use the word "correct" in test case names. Because it is hard to understand what it means, the next person looking at it (for example, a reviewer :D), would not know what is "correct" in this case.

```suggestion
  it('converts a list of filters to an expression AST node', () => {
```